### ISSUE TITLE
Clair does not install on latest Rascal version

### ISSUE THREAD

**Issue by grmbl99** [usethesource/clair#27]
Installing Clair on Eclipse with latest Rascal (0.23.0) fails with error:
```
Cannot complete the install because one or more required items could not be found.
  Software being installed: clair_feature 0.1.0.202104151358 (clair_feature.feature.group 0.1.0.202104151358)
  Missing requirement: clair 0.2.0.202103011036 (clair 0.2.0.202103011036) requires 'osgi.bundle; rascal_eclipse [0.17.0,0.19.0)' but it could not be found
  Cannot satisfy dependency:
    From: clair_feature 0.1.0.202104151358 (clair_feature.feature.group 0.1.0.202104151358)
    To: org.eclipse.equinox.p2.iu; clair [0.2.0.202103011036,0.2.0.202103011036]
```

**Comment by jurgenvinju** [comment]
Thanks Daan! Will look into this a.s.a.p. with @rodinaarssen

**Comment by jurgenvinju** [comment]
This is probably the "move to Java 11" step we had planned for Clair. am I right?

**Comment by jurgenvinju** [comment]
Working on this here: https://github.com/cwi-swat/clair/pull/31

The move to Java 11 and new Rascal API of 0.23.x seems to be without problems. We're currently working on compiling this "headless" again using Tycho which is always a bit of a challenge.

**Comment by jurgenvinju** [comment]
Compiled against latest Rascal en Tycho problems resolved. Next step is releasing a new plugin.

**Comment by jurgenvinju** [comment]
#32 was fixed, so now moving on to work on #33 , when that is fixed, this issue can be closed.

**Comment by jurgenvinju** [comment]
Almost there; waiting for https://github.com/usethesource/rascal/issues/1624
